static/js/recent_topics_ui.js

```diff
@@ -498,15 +508,47 @@ export function inplace_rerender(topic_key) {
     }
```

Comment:
I think we can significantly reduce the number of `complete_rerender` calls if this works well.